### PR TITLE
Added dynaprops for min and max attack range increase.

### PR DESCRIPTION
**Pull request by sigmantium**

### CODE REVIEW

File: docs/docs.polserver.com/pol100/configfiles.xml [hunk 1]
```diff
@@ -1286,110 +1286,113 @@ Region  (string regionname)
...
+      [LowerManaCost          (int/(die-string value))]
+      [HitChance              (int/(die-string value))]
+      [FireResistCap          (int/(die-string value))]
+      [ColdResistCap          (int/(die-string value))]
+      [EnergyResistCap        (int/(die-string value))]
+      [PhysicalResistCap      (int/(die-string value))]
+      [PoisonResistCap        (int/(die-string value))]
+      [Luck                   (int/(die-string value))]
+      [SwingSpeedIncrease     (int/(die-string value))]
+      [MinAttackRangeIncrease     (int/(die-string value))]
```
Comment (KevinEady) [inline docs/docs.polserver.com/pol100/configfiles.xml:1371]: Can we adjust the spacing of the existing mods to be even with the new ones so they are correctly aligned?

File: docs/docs.polserver.com/pol100/objref.xml [hunk 2]
```diff
@@ -551,11 +558,11 @@
 <member mname="memory_usage" type="Integer" access="r/o" mdesc="current process usage in KB" />
 <member mname="last_character_serial" type="Integer" access="r/o" mdesc="Last character serial number assigned by core" />
 <member mname="last_item_serial" type="Integer" access="r/o" mdesc="Last item serial number assigned by core" />
-
-<method proto="log_profile(bool clear)" returns="true/false" desc="Writes the script profile to the log, optionally clearing it after." />
-<method proto="set_priority_divide(int divide)" returns="true/false" desc="Sets the priority divide to 'divide'" />
-<method proto="clear_script_profile_counters()" returns="true/false" desc="Clears the script profile counters"/>
 <member mname="poldir" type="String" access="r/o" mdesc="Directory of the POL executable" />
```
Comment (KevinEady) [inline docs/docs.polserver.com/pol100/objref.xml:561]: Was this needed to be moved...?

File: docs/docs.polserver.com/pol100/configfiles.xml [hunk 3]
```diff
@@ -1286,110 +1286,113 @@ Region  (string regionname)
...
+      [LowerManaCostMod       (int/(die-string value))]
+      [HitChanceMod           (int/(die-string value))]
+      [FireResistCapMod       (int/(die-string value))]
+      [ColdResistCapMod       (int/(die-string value))]
+      [EnergyResistCapMod     (int/(die-string value))]
+      [PhysicalResistCapMod   (int/(die-string value))]
+      [PoisonResistCapMod     (int/(die-string value))]
+      [LuckMod                (int/(die-string value))]
+      [SwingSpeedIncreaseMod  (int/(die-string value))]
+      [AttackRangeIncreaseMod  (int/(die-string value))]
```
Comment (KevinEady) [inline docs/docs.polserver.com/pol100/configfiles.xml:1389]: I think this should be `MinAttackRangeIncreaseMod` and `MaxAttackRangeIncreaseMod` ...?

File: docs/docs.polserver.com/pol100/configfiles.xml [hunk 4]
```diff
@@ -1413,193 +1416,197 @@ NpcTemplate (string unique templatename)
...
-    AR               (int armor rating)
-    MaxHP            (int max)
-    [OnHitScript     (string script name)]
-    [Coverage        (string armor zone name {defaults to layer number zone})]
-    [Coverage...]
-}
-
-...and lots of any of these in the file...
-</structure>
+      (Class Name)  (integer unique 'objtype')
```
Comment (KevinEady) [inline docs/docs.polserver.com/pol100/configfiles.xml:1419]: more initial indentation changes

File: docs/docs.polserver.com/pol100/configfiles.xml (hunk 3 above)
Comment (KevinEady) [inline docs/docs.polserver.com/pol100/configfiles.xml:1389]: Alignment

File: docs/docs.polserver.com/pol100/configfiles.xml [hunk 5]
```diff
@@ -1286,110 +1286,113 @@ Region  (string regionname)
...
-    [PoisonResistCapMod     (int/(die-string value))]
-    [LuckMod                (int/(die-string value))]
-    [SwingSpeedIncreaseMod  (int/(die-string value))]
-
-    [CProp                  (string name, string value, will set a CProp on the npc at creation)]
-    [Cprop                  (...)]
-}
-[NpcTemplate...]
-</structure>
+      NpcTemplate (string unique templatename)
```
Comment (KevinEady) [inline docs/docs.polserver.com/pol100/configfiles.xml:1289]: I don't think it's necessary to change the initial indentation of the  `<structure>` texts. The HTML docs are rendered in a `<pre>` tag so they will have weird spaces.

File: docs/docs.polserver.com/pol100/configfiles.xml [hunk 6]
```diff
@@ -1413,193 +1416,197 @@ NpcTemplate (string unique templatename)
...
+      [HitChance              (int/(die-string value))]
+      [FireResistCap          (int/(die-string value))]
+      [ColdResistCap          (int/(die-string value))]
+      [EnergyResistCap        (int/(die-string value))]
+      [PhysicalResistCap      (int/(die-string value))]
+      [PoisonResistCap        (int/(die-string value))]
+      [Luck                   (int/(die-string value))]
+      [SwingSpeedIncrease     (int/(die-string value))]
+      [MinAttackRangeIncrease    (int/(die-string value))]
+      [MaxAttackRangeIncrease    (int/(die-string value))]
```
Comment (KevinEady) [inline docs/docs.polserver.com/pol100/configfiles.xml:1495]: alignment

File: docs/docs.polserver.com/pol100/configfiles.xml [hunk 7]
```diff
@@ -1413,193 +1416,197 @@ NpcTemplate (string unique templatename)
...
+      [HitChanceMod           (int/(die-string value))]
+      [FireResistCapMod       (int/(die-string value))]
+      [ColdResistCapMod       (int/(die-string value))]
+      [EnergyResistCapMod     (int/(die-string value))]
+      [PhysicalResistCapMod   (int/(die-string value))]
+      [PoisonResistCapMod     (int/(die-string value))]
+      [LuckMod                (int/(die-string value))]
+      [SwingSpeedIncreaseMod  (int/(die-string value))]
+      [MinAttackRangeIncreaseMod (int/(die-string value))]
+      [MaxAttackRangeIncreaseMod (int/(die-string value))]
```
Comment (KevinEady) [inline docs/docs.polserver.com/pol100/configfiles.xml:1513]: alignment

**Review by KevinEady — COMMENTED** [review]
Some comments mostly on the confgfile indentation + alignment

**Review by turleypol — APPROVED** [review]

**Pull request merged**